spec/checkPullRequestLabelsSpec.js

```diff
@@ -0,0 +1,269 @@
+// The plugin refers to the actual app in index.js.
+const oppiaBot = require('../index');
+const checkPullRequestLabelModule = require('../lib/checkPullRequestLabels');
+const scheduler = require('../lib/scheduler');
+let payload = require('../fixtures/pullRequestPayload.json');
+
+describe('Pull Request Label Check', () => {
+  /**
+   * @type {import('probot').Probot} robot
+   */
+  let robot;
+
+  /**
+   * @type {import('probot').Octokit} github
+   */
+  let github;
+
+  /**
+   * @type {import('probot').Application} app
+   */
+  let app;
+
+  beforeEach(() => {
+    spyOn(scheduler, 'createScheduler').and.callFake(() => {});
+
+    github = {
+      issues: {
+        createComment: jasmine.createSpy('createComment').and.returnValue({}),
+        addAssignees: jasmine.createSpy('addAssignees').and.resolveTo({}),
+      },
+    };
+
+    robot = createProbot({
+      id: 1,
+      cert: 'test',
+      githubToken: 'test',
+    });
+
+    app = robot.load(oppiaBot);
+    spyOn(app, 'auth').and.resolveTo(github);
+  });
+
+  describe('when pull request gets labeled', () => {
+    describe('assigns based on changelog label', () => {
+      const label = {
+        id: 638839900,
+        node_id: 'MDU6TGFiZWw2Mzg4Mzk5MDA=',
+        url: 'https://api.github.com/repos/oppia/oppia/labels/PR:%20released',
+        name: 'PR CHANGELOG: Server Errors -- @kevintab95',
+        color: '00FF00',
+      };
+      beforeEach(async () => {
+        // Set the payload action and label which will simulate adding
+        // the changelog label.
+        payload.payload.action = 'labeled';
+        payload.payload.label = label;
+        spyOn(checkPullRequestLabelModule, 'checkAssignee').and.callThrough();
+        await robot.receive(payload);
+      });
+
+      it('should call pull request label module', () => {
+        expect(checkPullRequestLabelModule.checkAssignee).toHaveBeenCalled();
+      });
+
+      it('should assign the project owner', () => {
+        expect(github.issues.addAssignees).toHaveBeenCalled();
+        const params = {
+          repo: payload.payload.repository.name,
+          owner: payload.payload.repository.owner.login,
+          number: payload.payload.number,
+          assignees: ['kevintab95'],
+        };
+        expect(github.issues.addAssignees).toHaveBeenCalledWith(params);
+      });
+      it('should ping the project owner', () => {
+        expect(github.issues.createComment).toHaveBeenCalled();
+        const params = {
+          repo: payload.payload.repository.name,
+          owner: payload.payload.repository.owner.login,
+          number: payload.payload.number,
+          body:
+            'Assigning @kevintab95 for the first-pass review' +
+            ' of this pull request. Thanks!',
+        };
+        expect(github.issues.createComment).toHaveBeenCalledWith(params);
+      });
+    });
+
+    it('does not assign project owner if he is the pr author', async () => {
```

Comment:
if he is the pr author -> if they are the pr author

Reply:
Done.